debian/patches/deprecation-version-boundary.patch

```diff
@@ -0,0 +1,20 @@
+Description: add deprecation boundary version
+This version makes any deprecations added after this version
+to be considered "new" deprecations and therefore get logged
+at an INFO log level and not cause exit 2.
+
+Author: Brett Holman <brett.holman@canonical.com>
+Last-Update: 2024-06-28
+---
+This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
+--- a/cloudinit/features.py
++++ b/cloudinit/features.py
+@@ -87,7 +87,7 @@ On Debian and Ubuntu systems, cc_apt_con
+ to write /etc/apt/sources.list directly.
+ """
+ 
+-DEPRECATION_INFO_BOUNDARY = "devel"
++DEPRECATION_INFO_BOUNDARY = "24.1"
```

Comment:
```suggestion
+DEPRECATION_INFO_BOUNDARY = "23.3"
```

Reply:
oops was looking at mantic-updates in rmadison  `cloud-init | 23.3.1-0ubuntu2                    | mantic          | source, all`